Update test build instructions in kernels readme

manuelcandales opened the pull request:
Differential Revision: D72420611

kirklandsign commented on kernels/README.md:
```diff
@@ -359,31 +359,12 @@ cmake . \
   -DEXECUTORCH_BUILD_DEVTOOLS=ON \
   -DEXECUTORCH_BUILD_VULKAN=OFF \
   -DEXECUTORCH_BUILD_XNNPACK=ON \
```
-DEXECUTORCH_BUILD_EXTENSION_RUNNER_UTIL=ON

kirklandsign replied:
Need to add that line

manuelcandales committed:
Update test build instructions in kernels readme (#9885)

Summary: Pull Request resolved: https://github.com/pytorch/executorch/pull/9885

Reviewed By: kirklandsign

Differential Revision: D72420611

manuelcandales commented:
@pytorchbot cherry-pick --onto release/0.6 -c docs
